### PR TITLE
beautifulsoup4 4.11.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/view/head:/CHANGELOG
License: https://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/view/head:/LICENSE
Requirements: https://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/view/head:/setup.py

Actions: 
1. Skip `py<36`
2. Remove `noarch: python`
3. Remove `req/build`
4. Fix `python` in host and run
5. Add missing `setuptools` and `wheel`
6. Move `tests` from the `beautifulsoup4` output to the higher level (inside the output tests skipped at all)
7. Add `pytest` to the `bs4` output
8. Improve `test`
9. Fix home and urls with HTTPS

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -42,19 +33,42 @@ outputs:
       run:
         - {{ pin_subpackage('beautifulsoup4', max_pin="x.x.x") }}
     test:
+      source_files:
+        - bs4/tests
```
Comment (chenghlee) [inline recipe/meta.yaml:38]: We probably don't want to use `source_files` in our tests; conda build's default behavior is to copy these files into the output packages, which among other things, can:
1. dramatically increase the size of the output package; and
2. occasionally trigger security warnings on user systems.

A better approach may be to create a `-tests` package as another output package, then use it as a test requires.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -42,19 +33,42 @@ outputs:
...
+      requires:
+        - pip
+        - pytest
+      commands:
+        - pip check
+        - pytest bs4 -vv  
+
+test:
+  source_files:
+    - bs4/tests
```
Comment (chenghlee) [inline recipe/meta.yaml:50]: See above comment about source files in the test section.

**Review by chenghlee — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: I've removed `source_files`

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: I've removed `source_files`

**Comment by skupr-anaconda** [comment]
Skip building on win32. 
tests failed only on win32:
```
Solving environment: ...working... failed
failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform noarch: {"beautifulsoup4[version='>=4.11.1,<4.11.2.0a0']"}
WARNING:conda_build.build:failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform noarch: {"beautifulsoup4[version='>=4.11.1,<4.11.2.0a0']"}
```

**Review by ScottBailey — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
Other than the issue with the source files everything else seems to be in order.

**Pull request merged**